integration_test/cases/browser/set_value_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Wallaby.Integration.Browser.SetValueTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "forms.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "set_value/3" do
+    test "accepts a query", %{page: page} do
+      page
+      |> send_keys(Query.text_field("Name"), ["Chris", :tab, "c@keathley.io"])
```

Comment:
I copied the test code from sends_keys and seem to have left this in as a hangover from that.  To be honest, I would not have a test for  "accepts a query", as I could not see what it was doing over and above the other tests. I kept it to stay consistent, but did not tailor it to set_value. Mea culpa.